src/Sylius/Bundle/UserBundle/Resources/config/app/config.yml

```diff
@@ -4,6 +4,7 @@
 security:
     password_hashers:
         argon2i: argon2i
+        plaintext: plaintext
```

Comment:
I'd be for removing this line from here and adding it only in test contexts to not have this hasher present by default.